src/AutoPilotPlugins/PX4/PX4FlightBehavior.cc

```diff
@@ -0,0 +1,86 @@
+/****************************************************************************
+ *
+ * (c) 2009-2020 QGROUNDCONTROL PROJECT <http://www.qgroundcontrol.org>
+ *
+ * QGroundControl is licensed according to the terms in the file
+ * COPYING.md in the root of the source code directory.
+ *
+ ****************************************************************************/
+
+
+#include "PX4FlightBehavior.h"
+#include "PX4AutoPilotPlugin.h"
+#include "AirframeComponent.h"
+
+PX4FlightBehavior::PX4FlightBehavior(Vehicle* vehicle, AutoPilotPlugin* autopilot, QObject* parent)
+    : VehicleComponent(vehicle, autopilot, parent)
+    , _name(tr("Flight Behavior"))
+{
+}
+
+QString PX4FlightBehavior::name(void) const
```

Comment:
Just a note, (void) is unnecessary in C++ code.

Reply:
copy-paste, and I agree